collect_app/src/main/java/org/fieldsight/naxa/v3/project/ProjectDashboardActivity.java

```diff
@@ -339,8 +433,23 @@ private void setupNavigationHeader() {
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         getMenuInflater().inflate(R.menu.menu_main, menu);
+        menu.findItem(R.id.action_refresh).setShowAsAction(SHOW_AS_ACTION_ALWAYS);
+
         return true;
     }
+
+    @Override
+    public boolean onPrepareOptionsMenu(Menu menu) {
+//        menu.findItem(R.id.action_refresh).setVisible(showSyncMenu);
+        if (syncStarts) {
+            menu.findItem(R.id.action_refresh).setVisible(false);
```

Comment:
change it to cancel 
1. change cancel icon
2. function to cancel call is in projectlistactivityv3